client/app/domain/domain.service.js

```diff
@@ -1314,6 +1314,22 @@ angular.module("services").service(
         }
 
         /**
+         * Get DNS Anycast status
+         * @param {string} serviceName
+         */
+        getDnsAnycast (serviceName) {
+            return this.OvhHttp
+                .get(`/domain/zone/${serviceName}`, { rootPath: "apiv6" })
+                .then((response) => {
+                    if (response.hasDnsAnycast != null) {
+                        return { status: (response.hasDnsAnycast && "enabled") || "disabled" };
```

Comment:
Not a big fan of this kind of thing, I feel like it's less readable than `response.hasDnsAnycast ? "enabled" : "disabled"`